react-charts/donut-chart/step5.md

```diff
@@ -1,52 +1,39 @@
-Now we'll add a legend to the donut chart...
+Change the donut chart theme color.
 
-## Task
+1) <strong>Make sure the App.js file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component and add a property to change the theme.</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
 &lt;ChartDonut
   constrainToVisibleArea
   data={[{ x: &#39;Cats&#39;, y: 35 }, { x: &#39;Dogs&#39;, y: 55 }, { x: &#39;Birds&#39;, y: 10 }]}
   labels={({ datum }) =&gt; `${datum.name}: ${datum.y}%`}
+  legendData={[
+    { name: &#39;Cats: 35&#39; }, 
+    { name: &#39;Dogs: 55&#39; }, 
+    { name: &#39;Birds: 10&#39; }
+  ]}
+  legendOrientation=&quot;vertical&quot;
+  legendPosition=&quot;right&quot;
   subTitle=&quot;Pets&quot;
   title=&quot;100&quot;
   height={230}
   width={350}
 /&gt;
 </pre>
 
-3) Replace the following property in that section:
+3) <strong>Add the 'themeColor' property to the component.</strong>
 
-<pre class="file" data-target="clipboard">
-padding={{
-  bottom: 20,
-  left: 20,
-  right: 140,
-  top: 20
-}}
-</pre>
-
-- This adds more padding to the right property to accommodate the legend.
+The `themeColor` property specifies the theme color. Some valid values are `ChartThemeColor.blue`,  `ChartThemeColor.green`, or `ChartThemeColor.multi`.

```

Comment:
@abigaeljamie  - are these quotes you are referring to that commas would go in?  These are markdown backticks for isolating code samples - it would be confusing to put commas in there.

Reply:
Ha, they looked like quotes to me. Disregard :)